draft-liu-teas-transport-network-slice-yang.md

```diff
@@ -147,9 +149,9 @@ author:
 
 ### Use Case 2 : 
 
-   The current expression of slice requests leveraging on {{?I-D.draft-ietf-teas-ietf-network-slice-nbi-yang}} allows the customer to request distinct connectivity constructs as part of the same Network Resource Partition (NRP). The topology provided by the customer could imply different NRPs, instead.
+   The current expression of slice requests leveraging on {{!I-D.ietf-teas-ietf-network-slice-nbi-yang}} allows the customer to request distinct connectivity constructs as part of the same Network Resource Partition (NRP). The topology provided by the customer could imply different NRPs, instead.
 
-   As an another example, from realization perspective even on the same NRP, a slice requests leveraging on {{?I-D.draft-ietf-teas-ietf-network-slice-nbi-yang}} without topology differentiation could imply the realization of all the connectivity constructs on the same manner. For instance, implementing all of them within the same VRF in a L3VPN. The usage of the topological views can help the provider to infer differentiated realization of some of the connectivity constructs, for instance, by implementing them on different VRFs. This can have operational advantages (e.g., adding new nodes / SDPs could affect / imply limit the necessary VRF reconfiguration only to the one including affected connectivity constructs).
+   As an another example, from realization perspective even on the same NRP, a slice requests leveraging on {{!I-D.ietf-teas-ietf-network-slice-nbi-yang}} without topology differentiation could imply the realization of all the connectivity constructs on the same manner. For instance, implementing all of them within the same VRF in a L3VPN. The usage of the topological views can help the provider to infer differentiated realization of some of the connectivity constructs, for instance, by implementing them on different VRFs. This can have operational advantages (e.g., adding new nodes / SDPs could affect / imply limit the necessary VRF reconfiguration only to the one including affected connectivity constructs).

```

Comment:
This sentence it seems to me a bit hard to read and understand. Is only my problem? The essence should be that operational advantage is the fact that reconfiguration would be needed only for impacted connectivity constructs .
Maybe we can simply summarized with these words.

Reply:
Suggested new text: "As an another example, a slice requests leveraging on {{!I-D.ietf-teas-ietf-network-slice-nbi-yang}} without topology differentiation could imply the realization of all the connectivity constructs on the same manner, even on the same NRP. For instance, assuming the realization of the slice by means of L3VPN, without a topological representation of separated constructs groupings, the implementing of all of them could be implemented on the same VRF. However, the usage of the topological views can help the provider to infer differentiated realization of some of the connectivity constructs, for instance, by implementing them on different VRFs. This kind of realization can be justified due to operational advantages (e.g., adding new nodes / SDPs could affect / imply limit the necessary VRF reconfiguration only to the one including affected connectivity constructs)."

Reply:
ok for me

Reply:
beautifying above paragraph with some little help from gen-ai resulted in the following:

As another example, a slice request leveraging {{!I-D.ietf-teas-ietf-network-slice-nbi-yang}} without topology differentiation could result in all connectivity constructs being realized in the same manner on the same NRP, e.g. implementing all of them within the same VRF in a L3VPN. Using topological views can help providers infer differentiated realizations of some of the connectivity constructs, for instance, by implementing them on different VRFs. This approach can offer operational advantages, like limiting the necessary VRF reconfiguration to only those affected connectivity constructs when adding new nodes or SDPs.